### PR TITLE
Bug fix - To do Draft for Voluntary Dissolution

### PR DESCRIPTION
**Pull request by patrickpeinanw**
*Issue:*https://github.com/bcgov/entity/issues/23569

*Description of changes:*
Fix the bug for a draft voluntary dissolution, to properly display the title and action button. Mismatch resolved.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-dashboard-dev--pr-56-vk88vcgm.web.app

File: src/utils/todo/task-filing/button-loader.ts [hunk 1]
```diff
@@ -108,14 +108,14 @@ const showDeleteOnly = (todoItem: TodoItemI): boolean => {
...
-    return true
-  } else if (filingType === FilingTypes.SPECIAL_RESOLUTION) {
-    // if a business is not in good standing, non-staff role can only delete the Special Resolution draft
-    return business && !business.currentBusiness.goodStanding && !isStaffAccount
-  } else {
-    return false
+
+  switch (filingType) {
+    case FilingTypes.ALTERATION:
+    case FilingTypes.DISSOLUTION:
```
Comment (hfekete) [inline src/utils/todo/task-filing/button-loader.ts:114]: Just a question, so for delete button show we treat both alteration and dissolution as if it is special resolution ?

**Review by hfekete — APPROVED** [review]
LGM with a question

File: src/utils/todo/task-filing/button-loader.ts (hunk 1 above)
Comment (patrickpeinanw) [reply src/utils/todo/task-filing/button-loader.ts:114]: Yes. The switch statement is copied from the old codebase. I misunderstood the logic before.

**Review by BrandonSharratt — APPROVED** [review]
Looks good to me

**Pull request merged**